config/cylc_hpc/flow.cylc

```diff
@@ -46,23 +53,27 @@
         script = """
             singularity exec --bind $soit_dir:/tmp $project_dir/fetchdata.simg python3 /usr/local/bin/pass_time_cylc.py --startdate $startdate --enddate $enddate --csvoutpath /tmp --centroid_x $centroid_x --centroid_y $centroid_y --SPACEUSER $SPACEUSER --SPACEPSWD $SPACEPSWD
         """
+    [[pulljuliaimage]]
+        script = """
+            apptainer build --force $project_dir/icefloetracker-julia.simg docker://brownccv/icefloetracker-julia:pr-45
+        """
```

Comment:
After merging, I'll have to start a new PR and change the image tag to `main` rather than `pr-45`.